Mathlib/CategoryTheory/FintypeCat.lean

```diff
@@ -198,5 +198,77 @@ lemma isSkeleton : IsSkeletonOf FintypeCat Skeleton Skeleton.incl where
   skel := Skeleton.is_skeletal
   eqv := by infer_instance
 
+section Universes
+
+universe v
+
+/-- If `u` and `v` are two arbitrary universes, we may construct a functor
+`switchUniverse.{u, v} : FintypeCat.{u} ⥤ FintypeCat.{v}` by sending
+`X : FintypeCat.{u}` to `ULift.{v} (Fin (Fintype.card X))`. -/
+noncomputable def switchUniverse : FintypeCat.{u} ⥤ FintypeCat.{v} where
```

Comment:
```suggestion
noncomputable def uLift : FintypeCat.{u} ⥤ FintypeCat.{v} where
```
is a bit shorter? Or is there a reason to avoid this name?

Reply:
Aaah, silly me... the universe doesn't have to increase...

Reply:
I am fine with `uSwitch`, although there is [CategoryTheory.uliftFunctor](https://leanprover-community.github.io/mathlib4_docs/Mathlib/CategoryTheory/Types.html#CategoryTheory.uliftFunctor) so maybe analogous to that it could be called `uswitchFunctor` but that is long again. What do you say? `uSwitch` or `uswitchFunctor`?